Competition2017/tests/competition/subsystems/drive/DriveTestBase.java

```diff
@@ -53,10 +53,60 @@ public void verifyDriveNegative() {
     public void setDriveEncoderDistances(double left, double right) {
         ((MockCANTalon)drive.leftDrive).setPosition(left);
         ((MockCANTalon)drive.rightDrive).setPosition(right);
+        
+        drive.updatePeriodicData();
+        pose.updatePeriodicData();
     }
     
     public void driveDeltaEncoderDistances(double left, double right){
-        ((MockCANTalon)drive.leftDrive).setPosition(((MockCANTalon)drive.leftDrive).getEncoderPosition() + left);
-        ((MockCANTalon)drive.rightDrive).setPosition(((MockCANTalon)drive.leftDrive).getEncoderPosition() + right);
+        setDriveEncoderDistances(
+                ((MockCANTalon)drive.leftDrive).getEncoderPosition() + left, 
+                ((MockCANTalon)drive.rightDrive).getEncoderPosition() + right);
+    }
+    
+    public void driveDeltaEncoderInches(double leftInches, double rightInches) {
+        driveDeltaEncoderDistances(
+                drive.convertInchesToTicks(leftInches),
+                drive.convertInchesToTicks(rightInches));
+    }
+    
+    public void verifyDriveArcingLeft(double minimumDifference) {
+        assertTrue((getRightSetpoint() - getLeftSetpoint()) > minimumDifference);
+    }
+    
+    public void verifyDriveArcingRight(double minimumDifference) {
+        assertTrue((getLeftSetpoint() - getRightSetpoint()) > minimumDifference);
+    }
+    
+    public void verifyDriveArcingRight() {
+        assertTrue(((MockCANTalon)drive.rightDrive).getSetpoint() < ((MockCANTalon)drive.leftDrive).getSetpoint());
+        assertTrue(((MockCANTalon)drive.leftDrive).getSetpoint() > 0);
+    }
+    
+    public void verifyTurningLeft() {
+        assertTrue(getLeftSetpoint() < 0);
+        assertTrue(getRightSetpoint() > 0);
+    }
+    
+    public void verifyTurningRight() {
+        assertTrue(getLeftSetpoint() > 0);
+        assertTrue(getRightSetpoint() < 0);
+    }
+    
+    public void verifyNotTurning(double minimumDifference) {
+        assertTrue((getLeftSetpoint() - getRightSetpoint()) < minimumDifference);
```

Comment:
Using assertEquals also has some nice advantages like when this fails it'll print out what the values were as opposed to just saying an assertion failed